viewing_party/main.py

```diff
@@ -0,0 +1,154 @@
+def create_movie(title, genre, rating):
+    movie_to_watch = {}
+    if title and genre and rating:
+        movie_to_watch["title"] = title
+        movie_to_watch["genre"] = genre
+        movie_to_watch["rating"] = rating
+    else:
+        return None
+    
+    return movie_to_watch  
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    watchlist = user_data["watchlist"]
+    
+    for i, movie in enumerate(watchlist):
+        if movie["title"] == title:
+            user_data["watched"].append(movie)
+            del watchlist[i]
```

Comment:
In general be careful with this kind of loop and the `del` or `remove` method.  This is because if  you are intending to delete most movies, this kind of loop will lead to potentially skipping an entry.

example if `watchlist = [{ "title": "Rocky"}, { "title": "Rocky"}, { "title": "Star Wars" }]` and `title = "Rocky"` will result in 

`[{ "title": "Rocky"}, { "title": "Star Wars" }]` because it deletes the 1st entry of Rocky and the other elements shift left 1 entry and the if statement never looks at the 2nd Rocky entry.